### ISSUE TITLE
With Hacktoberfest 2018 ending, will this repo be shut down? (No!) Can I still contribute? (Yes!)

### ISSUE THREAD

**Issue by hackleygithub** [wzhouwzhou/one-line-wonders#259]

**Comment by wzhouwzhou** [comment]
No, this repository will not be shut down, and you can still contribute after the end of Hacktoberfest2018. For anyone still interested in getting a free shirt for Hacktoberfest, however, you still have over a day left as Pull Requests count through October 31st, and you can view the countdown at this repository's website: https://hacktoberfest2018.suya.moe

**Comment by wzhouwzhou** [comment]
🙈

**Comment by wzhouwzhou** [comment]
Hactoberfest 2019 is around the corner, happy coding!
